src/app/v2/inventory/projects/columns.tsx

```diff
@@ -29,6 +30,16 @@ export const columns: ColumnDef<Project>[] = [
   {
     accessorKey: "name",
     header: "Name",
+    cell: ({ row }) => {
+      return (
+        <Link href={`/v2/inventory/projects/${row.original.id}`}>
+          <div className="flex items-center gap-1">
+            <span>{row.original.name}</span>
+            <ArrowUpRight className="h-4 w-4" />
+          </div>
+        </Link>
+      );
+    },
   },
```

Comment:
Add link to project details in the `View Details` drop down

![Uploading Screenshot 2025-02-04 at 4.05.42 PM.png…]()